fvm/bootstrap.go

```diff
@@ -166,12 +166,15 @@ func (b *BootstrapProcedure) createServiceAccount(accountKey flow.AccountPublicK
 func (b *BootstrapProcedure) deployFungibleToken() flow.Address {
 	fungibleToken := b.createAccount()
 
-	err := b.vm.invokeMetaTransaction(
+	txError, err := b.vm.invokeMetaTransaction(
 		b.ctx,
 		deployContractTransaction(fungibleToken, contracts.FungibleToken(), "FungibleToken"),
 		b.sth,
 		b.programs,
 	)
+	if txError != nil {
```

Comment:
I think that It might actually make sense to define a local `func panicOnMetaInvokeErr(errorDecsription, txError, err)` to remove some duplicate code.

Reply:
applied at be2e6e8